tests/baselines/reference/jsDeclarationsInterfaces.js

```diff
@@ -135,10 +135,10 @@ export interface B {
 export interface C<T_1, U_1> {
     new (): string;
     new (x: T_1): U_1;
-    new <Q_4>(x: Q_4): T_1 & Q_4;
+    new <Q_6>(x: Q_6): T_1 & Q_6;
     (): number;
     (x: T_1): U_1;
-    <Q_3>(x: Q_3): T_1 & Q_3;
+    <Q_4>(x: Q_4): T_1 & Q_4;

```

Comment:
Not 100% sure what triggered these changes, but seems fine?

Reply:
The intervening number got marked as potentially used when traversing the base classes, but then ultimately elided, I believe. I could put in some extra effort to keep the numbers compressed, but because it doesn't particularly matter, I saw no reason to.